server/publications.js

```diff
@@ -5,6 +5,10 @@ Meteor.publish(null, function () {
     return [
       Users.find({
         _id: this.userId
+      }, {
```

Comment:
probably don't want to use 'fields' here - this will make it so that only _id and interestedInJobIds are sent from the server, but we want all fields - should be able to just remove this